### PR TITLE
Example twitter dag

### PR DESCRIPTION
**Pull request by ekhtiar**
Added an example dag "example_twitter_dag" and a readme markdown "example_twitter.md" explaining the example dag. Also a python_script folder was added under the example_dags folder to hold python scripts for this example, and other examples in future.

### CODE REVIEW

**Comment by jackgolding** [comment]
Nice write up!

**Comment by mistercrunch** [comment]
Oh nice, I think it's our first non-abstract example! Can you please rebase, I think you may have branched off while landscape was broken...

File: airflow/example_dags/example_twitter.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Example Twitter DAG
+
+<p><strong>Introduction: </strong>This example dag depicts a typical ETL process and is a perfect use case automation scenario for Airflow. <i>Please note that the main scripts associated with the tasks are returning None.</i> The purpose of this DAG is to demonstrate how to write a functional DAG within Airflow.</p>
```
Comment (mistercrunch) [inline airflow/example_dags/example_twitter.md:3]: I'm surprised to see raw html in markdown, strong would usually be something like `**this is bold**`

File: airflow/example_dags/example_twitter_dag.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+from airflow.operators import PythonOperator
+from datetime import datetime, timedelta
+from airflow.hooks import HiveCliHook
+from datetime import date, timedelta
+
+#-------------------------------------------------------------------------------------------------------------------------------------
+# Import Scripts for Python callable
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+sys.path.append("python_scripts/example_twitter/")
```
Comment (mistercrunch) [inline airflow/example_dags/example_twitter_dag.py:26]: it's usually pretty bad to mess with `sys.path`, use relative or absolute imports

File: airflow/example_dags/example_twitter_dag.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+#-------------------------------------------------------------------------------------------------------------------------------------
+# Load The Dependencies
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+import sys
+reload(sys)
+sys.setdefaultencoding("utf-8")
+import time
+from airflow import DAG
+from airflow.operators import BashOperator
```
Comment (mistercrunch) [inline airflow/example_dags/example_twitter_dag.py:15]: 15 to 17 could be a one liner with comas

File: airflow/example_dags/example_twitter_dag.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    # 'queue': 'bash_queue',
+    # 'pool': 'backfill',
+    # 'priority_weight': 10,
+    # 'end_date': datetime(2016, 1, 1),
+}
+
+dag = DAG(
+	'example_twitter_dag', default_args = default_args,
```
Comment (mistercrunch) [inline airflow/example_dags/example_twitter_dag.py:53]: I think there are tabs in here, no tabs pls.

**Comment by mistercrunch** [comment]
You should set up landscape against your fork and rebase, you'll get some linting error to fix. This repo is now pretty strict about PEP8 and other coding standards, make the code feel more uniform.

**Comment by ekhtiar** [comment]
Thanks for the comments and compliments.
Really loving the Airflow community! I will rebase, make those changes, and open a new pull request. Thank you once again max, you rock!

**Comment by mmacny** [comment]
Wanted to comment on the extreme value of examples like this. I've got no skin in the game for this PR, but this would def. help some newcomers to Hive+Airflow in my team understand incremental tasks better. Thanks guys!

File: airflow/example_dags/example_twitter.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+<p>As most probably these folders and hive tables doesn't exist in your system, you will get an error for these tasks within the DAG. If you rebuild a function DAG from this example, make sure those folders and hive tables exists. When you create the table, keep the consideration of table partitioning and declaring comma as the row deliminator in your mind. Furthermore, you may also need to skip headers on each read and ensure that the user under which you have Airflow running has the right permission access. Below is a sample HQL snippet on creating such table:</p>
+```
+CREATE TABLE toTwitter_A(id BIGINT, id_str STRING<br/>
```
Comment (underyx) [inline airflow/example_dags/example_twitter.md:41]: I think that `<br/>` should probably not be present in this code block.

Comment (ekhtiar) [inline airflow/example_dags/example_twitter.md:41]: Thank you for pointing our the <br/>

**Comment by ekhtiar** [comment]
@mmacny Thanks a lot! I will be sure to write up a few other ones like these using real life use cases and latest technologies available in the big data ecosystem :D

**Comment by bolkedebruin** [comment]
This has been merge in a separate PR.

**Pull request closed without merging**